simtools/io_operations/hdf5_handler.py

```diff
@@ -106,7 +106,7 @@ def read_hdf5(hdf5_file_name):
 
     with tables.open_file(hdf5_file_name, mode="r") as file:
         for node in file.walk_nodes("/", "Table"):
-            table_path = node._v_pathname
+            table_path = node._v_pathname  # pylint: disable=protected-access
```

Comment:
Disabled pylint error because it seems it is not actually a protected member but just a reserved prefix. At least I could not find a way to get it in another way. It should be noted that ctapipe access this variable as well.

Reply:
Agree.